identity-iota/src/did/doc/iota_document.rs

```diff
@@ -388,6 +393,18 @@ impl IotaDocument {
     Ok(())
   }
 
+  pub fn attach_method_relationship(&mut self, did_url: IotaDIDUrl, relationship: MethodRelationship) -> Result<bool> {
+    let core_did_url: CoreDIDUrl = CoreDIDUrl::from(did_url);
+    let was_attached = self.document.attach_method_relationship(core_did_url, relationship)?;
+    Ok(was_attached)
+  }
+
+  pub fn detach_method_relationship(&mut self, did_url: IotaDIDUrl, relationship: MethodRelationship) -> Result<bool> {
```

Comment:
Nit: doc comment public functions.

```suggestion
  /// Detaches the given relationship from the given method, if the method exists.
  pub fn detach_method_relationship(&mut self, did_url: IotaDIDUrl, relationship: MethodRelationship) -> Result<bool> {
```